### DISCUSSION TITLE
The sector transfer directory command does not work?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9401] [Storage Provider] [answered]
I can't use "lotus miner storage redeclare -- id s-t01862564-337" or "lotus miner storage redeclare -- id 337". What's the problem?

Step 1: Copy the file to the new directory.

Step 2: "lotus miner storage redemption -- id" lotus miner storage redemption -- id s-t01862564-337 "

Step 3: "lotus miner storage redemption -- id 337"

**Comment by rjan90** [comment, marked as answer]
Hey @ChaiWenyu1983! The `--id` is for the storage path id (which you can find with `lotus-miner storage list`), and not the sector-id 😄 

So to redeclare the sector in the new path you have to:
`lotus-miner storage redeclare --id <storage-path-id>`

**Reply by ChaiWenyu1983** [reply]
Excuse me，Is the id“337” or “s-t01862564-337“？
How do I know if the execution is successful?

**Reply by rjan90** [reply]
You need to specify the id for the storage path, and not the sector. You can find the storage path id in: `lotus-miner storage list`.

```
lotus-miner storage list
16fc36d4-ab86-495e-a6d2-003a3a776f15:
        [#####################################             ] 328.9 GiB/436.3 GiB 75%
        Unsealed: 1; Sealed: 49; Caches: 49; Updated: 0; Update-caches: 0; Reserved: 0 B
        Weight: 10; Use: Seal Store
        Local: /root/.lotus-miner-local-net
        URL: http://127.0.0.1:2345/remote

abf69211-59f7-4607-8a5c-4def54c55f25:
        [#####################################             ] 328.9 GiB/436.3 GiB 75%
        Unsealed: 0; Sealed: 2; Caches: 2; Updated: 0; Update-caches: 0; Reserved: 0 B
        Use: ReadOnly
        Local: /root/.genesis-sectors
        URL: http://127.0.0.1:2345/remote
```

This `16fc36d4-ab86-495e-a6d2-003a3a776f15` and `abf69211-59f7-4607-8a5c-4def54c55f25` are the ids for the storage paths.

You should be able to see that the sectors location is updater with `lotus-miner sector find 337` after executing the command.
